### DISCUSSION TITLE
win11 pcl2 启动动画不会显示

### DISCUSSION THREAD

**Discussion by 212112111zwl** [Meloong-Git/PCL#1009] [提问] [answered]
我刚升级到win11，打开pcl时过一会儿pcl窗口就出来了，但是没有启动的动画
快照版和发布版都没有，不知道为什么
各位win11有没有遇到过类似问题，或者能帮助我解决吗？

[Log2.txt](https://github.com/Hex-Dragon/PCL2/files/10497904/Log2.txt)
[Log3.txt](https://github.com/Hex-Dragon/PCL2/files/10497905/Log3.txt)
[Log4.txt](https://github.com/Hex-Dragon/PCL2/files/10497907/Log4.txt)
[Log5.txt](https://github.com/Hex-Dragon/PCL2/files/10497908/Log5.txt)
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/10497911/Log1.txt)

**Comment by WForst-Breeze** [comment]
弹窗动画是 Snapshot 2.4.5 更新内容，正式版 Release 2.4.4 是没有的（）至于快照版没有的话我不太清楚，您或许可以去 设置-启动器-调试选项-动画速度 看看……？qwq

**Reply by 212112111zwl** [reply]
正式版有放大动画，也不会显示

**Reply by 212112111zwl** [reply]
我将动画速度调到最慢，启动动画显示了，基本上说明不是pcl的问题

**Comment by 212112111zwl** [comment]
顺便说一下，启动后的动画都很正常，关闭时也很丝滑

**Comment by 212112111zwl** [comment, marked as answer]
已自行解决
我的硬件比较烂，pcl启动后任务栏会执行弹出pcl图标的动画，占用了太多资源，因而pcl窗口的启动动画不能正常显示
所以我把pcl固定到任务栏……就……好了……

**Comment by miner233** [comment]
下次有问题建议发 issues 而不是发 Discussions...
已经解决了就好

**Reply by 212112111zwl** [reply]
行，但是我win10时候是好的，升级到win11就不好了，我怀疑是我自己的问题，不是pcl的bug，所以就选了提问
